src/pages/Home.tsx

```diff
@@ -42,24 +45,33 @@ export const Home = () => {
     }
   }, [token]);
 
+  console.log(tokenStore.statusResult);
+  console.log(ballot.ballotLoading);
```

Comment:
You missed some console.log statements here.

Reply:
I recommend GitHubs suggestion feature for such things.
See https://github.com/LeoTuet/ui-we-want-to-sleep/pull/70#discussion_r835096363